ui/packages/ui/src/index.html

```diff
@@ -14,6 +14,7 @@
 <body>
   <noscript>Enabling JavaScript is required to run this app.</noscript>
   <div id="root"></div>
+  <script src="config.js"></script>
```

Comment:
1. Should I make this an endpoint in the Backend that returns the right config? What happens if the file can't be loaded?

2. Don't we have a timing issue here when someone starts using the React app before that config.js is returned? Maybe move this to the `<head>` to avoid such timing issues?

Reply:
1. Yes, It needs to generate ‘config.js’ with required configuration. If file doesn’t exist or can’t be loaded. It will take default configuration which is under `ui/packages/services/src/config/config.service.ts`
2.  App bundle loads after ‘config.js’ so it shouldn’t be an issue.

Reply:
The 404 is an issue. People running the container and monitoring it would not like to see 404 in their logs. I fixed that by having the backend always return config. Is there any way we can remove the `mode` property and the other stuff from the response?
I will log an issue for cleaning up that feature and also fix the possible timing issue, because:

(2): No, that's obviously not a problem, but the other way around. The app is already loaded but `config.js` did not return, then it will use the default config.